src/Uno.UWP/Globalization/NumberFormatting/NumeralSystemTranslatorHelper.cs

```diff
@@ -0,0 +1,3566 @@
+		private static readonly char[] MongDigits = new char[] { '\u1810', '\u1811', '\u1812', '\u1813', '\u1814', '\u1815', '\u1816', '\u1817', '\u1818', '\u1819' };
+		private static readonly char[] MteiDigits = new char[] { '\uABF0', '\uABF1', '\uABF2', '\uABF3', '\uABF4', '\uABF5', '\uABF6', '\uABF7', '\uABF8', '\uABF9' };
+		private static readonly char[] MymrDigits = new char[] { '\u1040', '\u1041', '\u1042', '\u1043', '\u1044', '\u1045', '\u1046', '\u1047', '\u1048', '\u1049' };
+		private static readonly char[] MymrShanDigits = new char[] { '\u1090', '\u1091', '\u1092', '\u1093', '\u1094', '\u1095', '\u1096', '\u1097', '\u1098', '\u1099' };
+		private static readonly char[] NkooDigits = new char[] { '\u07C0', '\u07C1', '\u07C2', '\u07C3', '\u07C4', '\u07C5', '\u07C6', '\u07C7', '\u07C8', '\u07C9' };
+		private static readonly char[] OlckDigits = new char[] { '\u1C50', '\u1C51', '\u1C52', '\u1C53', '\u1C54', '\u1C55', '\u1C56', '\u1C57', '\u1C58', '\u1C59' };
+		private static readonly char[] OryaDigits = new char[] { '\u0B66', '\u0B67', '\u0B68', '\u0B69', '\u0B6A', '\u0B6B', '\u0B6C', '\u0B6D', '\u0B6E', '\u0B6F' };
+		private static readonly char[] SaurDigits = new char[] { '\uA8D0', '\uA8D1', '\uA8D2', '\uA8D3', '\uA8D4', '\uA8D5', '\uA8D6', '\uA8D7', '\uA8D8', '\uA8D9' };
+		private static readonly char[] SundDigits = new char[] { '\u1BB0', '\u1BB1', '\u1BB2', '\u1BB3', '\u1BB4', '\u1BB5', '\u1BB6', '\u1BB7', '\u1BB8', '\u1BB9' };
+		private static readonly char[] TaluDigits = new char[] { '\u19D0', '\u19D1', '\u19D2', '\u19D3', '\u19D4', '\u19D5', '\u19D6', '\u19D7', '\u19D8', '\u19D9' };
+		private static readonly char[] TamlDecDigits = new char[] { '\u0BE6', '\u0BE7', '\u0BE8', '\u0BE9', '\u0BEA', '\u0BEB', '\u0BEC', '\u0BED', '\u0BEE', '\u0BEF' };
+		private static readonly char[] TeluDigits = new char[] { '\u0C66', '\u0C67', '\u0C68', '\u0C69', '\u0C6A', '\u0C6B', '\u0C6C', '\u0C6D', '\u0C6E', '\u0C6F' };
+		private static readonly char[] ThaiDigits = new char[] { '\u0E50', '\u0E51', '\u0E52', '\u0E53', '\u0E54', '\u0E55', '\u0E56', '\u0E57', '\u0E58', '\u0E59' };
+		private static readonly char[] TibtDigits = new char[] { '\u0F20', '\u0F21', '\u0F22', '\u0F23', '\u0F24', '\u0F25', '\u0F26', '\u0F27', '\u0F28', '\u0F29' };
+		private static readonly char[] VaiiDigits = new char[] { '\uA620', '\uA621', '\uA622', '\uA623', '\uA624', '\uA625', '\uA626', '\uA627', '\uA628', '\uA629' };
+
+		public static string GetNumeralSystem(string languageTag)
+		{
+			languageTag = languageTag.ToLowerInvariant();
+
+			switch (languageTag)
+			{
+				case "aa":
+					return "Latn";
+				case "aa-dj":
+					return "Latn";
+				case "aa-er":
+					return "Latn";
+				case "aa-et":
+					return "Latn";
+				case "af":
+					return "Latn";
+				case "af-na":
+					return "Latn";
+				case "af-za":
+					return "Latn";
+				case "agq":
+					return "Latn";
+				case "agq-cm":
+					return "Latn";
```

Comment:
Have you got this list from somewhere?

Reply:
There is a link in CultureInfo class documentation that shows supported languages by windows. "see the Language tag column in the list of language/region names supported by Windows. Culture names follow the standard defined by BCP 47"
[The link](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)

Reply:
Let's add this link in the code comments, for future updates.